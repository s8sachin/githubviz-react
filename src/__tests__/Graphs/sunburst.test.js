import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount, render, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import SunburstWithTooltips from '../../components/Graphs/sunburstchart';

configure({ adapter: new Adapter() });

describe('SunburstWithTooltips', () => {
it('should mount sunburst component', () => {
    const wrapper = shallow(< SunburstWithTooltips />)
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})