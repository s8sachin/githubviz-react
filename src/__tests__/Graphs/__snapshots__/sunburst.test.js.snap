// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SunburstWithTooltips should mount Header component 1`] = `
<Sunburst
  className=""
  colorType="literal"
  data={
    Object {
      "children": Array [
        Object {
          "children": Array [
            Object {
              "bigness": 1,
              "children": Array [],
              "name": "excellent",
            },
            Object {
              "bigness": 1,
              "children": Array [],
              "name": "chart",
            },
          ],
        },
        Object {
          "bigness": 1,
          "children": Array [],
          "labelStyle": Object {
            "fontSize": 15,
            "fontWeight": "bold",
          },
          "name": "cool",
        },
        Object {
          "bigness": 1,
          "children": Array [],
          "name": "dogs",
        },
        Object {
          "bigness": 1,
          "children": Array [],
          "name": "sunglasses",
        },
        Object {
          "children": Array [
            Object {
              "bigness": 1,
              "children": Array [],
              "name": "great",
            },
            Object {
              "bigness": 1,
              "children": Array [],
              "name": "label",
            },
          ],
        },
      ],
    }
  }
  getAngle={[Function]}
  getAngle0={[Function]}
  getColor={[Function]}
  getLabel={[Function]}
  getSize={[Function]}
  height={300}
  hideRootNode={false}
  margin={
    Object {
      "bottom": 50,
      "left": 50,
      "right": 50,
      "top": 50,
    }
  }
  onValueMouseOut={[Function]}
  onValueMouseOver={[Function]}
  style={
    Object {
      "stroke": "#fff",
    }
  }
  width={350}
/>
`;

exports[`SunburstWithTooltips should mount sunburst component 1`] = `
<Sunburst
  className=""
  colorType="literal"
  data={
    Object {
      "children": Array [
        Object {
          "children": Array [
            Object {
              "bigness": 1,
              "children": Array [],
              "name": "excellent",
            },
            Object {
              "bigness": 1,
              "children": Array [],
              "name": "chart",
            },
          ],
        },
        Object {
          "bigness": 1,
          "children": Array [],
          "labelStyle": Object {
            "fontSize": 15,
            "fontWeight": "bold",
          },
          "name": "cool",
        },
        Object {
          "bigness": 1,
          "children": Array [],
          "name": "dogs",
        },
        Object {
          "bigness": 1,
          "children": Array [],
          "name": "sunglasses",
        },
        Object {
          "children": Array [
            Object {
              "bigness": 1,
              "children": Array [],
              "name": "great",
            },
            Object {
              "bigness": 1,
              "children": Array [],
              "name": "label",
            },
          ],
        },
      ],
    }
  }
  getAngle={[Function]}
  getAngle0={[Function]}
  getColor={[Function]}
  getLabel={[Function]}
  getSize={[Function]}
  height={300}
  hideRootNode={false}
  margin={
    Object {
      "bottom": 50,
      "left": 50,
      "right": 50,
      "top": 50,
    }
  }
  onValueMouseOut={[Function]}
  onValueMouseOver={[Function]}
  style={
    Object {
      "stroke": "#fff",
    }
  }
  width={350}
/>
`;
